src/som/VM.java

```diff
@@ -339,8 +339,11 @@ public void initalize(final SomLanguage lang) throws IOException {
     mainActor = Actor.createActor(this);
     vmMirror = objectSystem.initialize();
 
-    if (VmSettings.ACTOR_TRACING) {
-      ActorExecutionTrace.recordMainActor(mainActor, objectSystem);
+    if (VmSettings.ACTOR_TRACING || VmSettings.MEDEOR_TRACING) {
```

Comment:
`MEDEO_TRACING` should probably be `KOMPOS_TRACING`.

Any idea for a name for your stuff?
If no, perhaps simply `RECORD_REPLAY_TRACING`?